store/modules/advocates.ts

```diff
@@ -35,28 +41,51 @@ export {
   Advocate
 }
 
-export default {
-  state () {
-    return {
-      advocates: []
-    }
-  },
-  getters: {
-    filteredAdvocates (state: any) {
-      const { advocates } = state
+export class State {
+  advocates: Advocate[] = []
+  regionFilters: string[] = []
+}
 
-      return advocates
-    }
+const actions = <ActionTree<State, any>> {
+  async fetchAdvocates ({ commit }): Promise<void> {
+    const advocatesModule = await import('~/content/advocates/advocates.json')
+    const advocates = advocatesModule.default || []
+    commit('setAdvocates', advocates)
   },
-  mutations: {
-    setAdvocates (state: any, payload: any) {
-      state.advocates = payload
-    }
+
+  updateRegionFilters ({ commit }, regionFilters: string[]): void {
+    commit('setRegionFilters', regionFilters)
+  }
+}
+
+const mutations = <MutationTree<State>> {
+  setAdvocates (state, advocates: Advocate[]) {
+    state.advocates = advocates
   },
-  actions: {
-    async fetchAdvocates () {
-      const advocatesModule = await import('~/content/advocates/advocates.json')
-      return advocatesModule.default || []
+
+  setRegionFilters (state, regionFilters: string[]) {
+    state.regionFilters = regionFilters
+  }
+}
+
+const getters = <GetterTree<State, any>> {
+  /**
+   * List of advocates filtered by selected regions.
+   * If no region is selected to filter, all advocates shall be shown.
+   */
+  filteredAdvocates ({ advocates, regionFilters }): Advocate[] {
+    if (regionFilters.length === 0) {
```

Comment:
One of our goals with the code is writing it in a way that you don't need to add a comment to describe what it does. If instead of having:
```
if (regionFilters.length === 0) {
...
```
you write
```
const noRegionFilters = regionFilters.length === 0

if (noRegionFilters) {
...
```
like we did [here](https://github.com/Qiskit/qiskit.org/blob/master/store/modules/events.ts#L130), your code is easy to read and you don't need the comment before the getter

Reply:
Good catch! I adjusted it to follow this.

Thanks @y4izus 🙂